Comb_Sort/comb_sort.js

```diff
@@ -0,0 +1,34 @@
+/* 
+Function to implement Comb sort using JavaScript
+Example : If input array is [9,7,8,3,2], then after implementing this code the output array will be [2,3,7,8,9].  
+*/
+
+function combSort(arr)
+{
+    // the shrink factor is taken to be 1.3
+    var gap = Math.floor(arr.length / 1.3);
+    
+    /* loop to reduce gap and implement sorting */ 
+    while(gap > 0){
```

Comment:
Either follow
```
keyword()
{
```
this or 
```
keyword() {
```

You have used the combination of both.

If you want to use the second option, then add a space between `)` and `{`